app/helpers/checkout_helper.rb

```diff
@@ -55,7 +55,6 @@ def display_line_item_tax_rates(line_item)
 
   def display_adjustment_tax_rates(adjustment)
```

Comment:
I've done it on the Adjustment#tax_rates method as this one is pretty straightforward and calls the former!